src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -122,31 +145,44 @@
                  (vec))
       (and to?
            routes-found?
-           (< (count updated-network-values) available-networks-count))
+           (< (count network-values-with-not-available-chains) available-networks-count))
       (conj {:type :add}))))
 
 (defn loading-network-amounts
-  [valid-networks disabled-chain-ids receiver-networks to?]
-  (let [disabled-set            (set disabled-chain-ids)
-        receiver-networks-set   (set receiver-networks)
-        receiver-networks-count (count receiver-networks)
-        valid-networks          (concat valid-networks disabled-chain-ids)]
+  [{:keys [valid-networks disabled-chain-ids receiver-networks token-networks-ids to?]}]
+  (let [disabled-set               (set disabled-chain-ids)
+        receiver-networks-set      (set receiver-networks)
+        receiver-networks-count    (count receiver-networks)
+        token-networks-ids-set     (set token-networks-ids)
+        valid-networks-set         (set valid-networks)
+        not-available-networks     (if to?
+                                     (filter #(not (token-networks-ids-set %)) receiver-networks)
+                                     [])
+        not-available-networks-set (set not-available-networks)
+        valid-networks             (concat valid-networks
+                                           disabled-chain-ids
+                                           (when to?
+                                             (filter #(not (valid-networks-set %))
+                                                     not-available-networks)))]
     (cond-> (->> valid-networks
                  (map (fn [k]
-                        (cond-> {:chain-id k
-                                 :type     (if (or to?
-                                                   (not (contains? disabled-set k)))
-                                             :loading
-                                             :disabled)}
+                        (cond->
+                          {:chain-id k
+                           :type     (cond
+                                       (contains? not-available-networks-set k)   :not-available
+                                       (or to?
+                                           (not (contains? disabled-set k)))      :loading
+                                       (and (not to?) (contains? disabled-set k)) :disabled)}
                           (and (not to?) (contains? disabled-set k))
                           (assoc :total-amount (money/bignumber "0")))))
                  (sort-by (fn [item]
                             (get network-priority-score
                                  (network-utils/id->network (:chain-id item)))))
                  (filter
-                  #(or (and to? (contains? receiver-networks-set (:chain-id %)))
-                       (and (not to?)
-                            (not (contains? disabled-chain-ids (:chain-id %))))))
+                  (fn [%]
```

Comment:
can you name this var? I guess it's network? 🤔

Reply:
Sure, leftover

Reply:
Updated!